corehq/apps/reports/v2/models.py

```diff
@@ -7,6 +7,7 @@
 
 
 EndpointContext = namedtuple('EndpointContext', 'slug urlname')
+ColumnContext = namedtuple('ColumnContext', 'title slug width')
```

Comment:
super valid point. I was coming from the perspective it's much easier to handle widths in pixels with drag/drop resize functionality I will definitely think about this when I get to working on the column resize bit. 

I have it in the code because I think (eventually) it would be nice to be able to save the column width states in a report (perhaps they are percentages instead of pixels) to recall later.